pipelines/base/enterprise-contract.yaml

```diff
@@ -0,0 +1,57 @@
+# The purpose of this pipeline is to execute the verify-enterprise-contract-v2 task for container
+# images that are built but not automatically released in order to provide early feedback to users.
+# When auto release is enabled, the task is executed by the release pipeline immediately after the
+# container images are built, thus it is not necessary to execute the task via this pipeline.
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: enterprise-contract
+  labels:
+    "skip-hacbs-test": "true"
+spec:
+  params:
+    - name: IMAGES
+      type: string
+      description: |
+        Spec section of an ApplicationSnapshot. The containerImage of each
+        component will be verified.
```

Comment:
Hmm. It is parsed as an ApplicationSnapshot Spec by the ec-cli. I don't know if we want to hide that from users as it may cause confusion (maybe?). How about adding an example to the description?

Reply:
Updated the description. Let me know what you think.